app/views/posts/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<!-- app/views/posts/new.html.erb -->
+<h1>New Post</h1>
+
+<%= render 'form', post: @post %>
+
+<%= link_to 'Back', posts_path %>
```

Comment:
- [ ] Great progress on the form implementation! 👏 Your effort is evident. I'd like to suggest a slight improvement in the structure of your routes, specifically for the **posts form**.

    - Consider creating a dedicated route for the posts form, such as `/posts/new`. Currently, you're rendering the form on `/users/:user_id/`, which is intended for viewing user details. Separating the post form onto its own route enhances the organization and clarity of your application.

    - After successfully submitting a post, it's good practice to redirect users to the user details page (`/users/:user_id/`). This provides a seamless user experience, allowing them to see their posts in the context of the post. See below for more details on your UI:
<img width="1072" alt="Screenshot 2023-11-10 at 10 45 20" src="https://github.com/iankiema/BlogApp/assets/107192076/665ac5ce-d6de-435d-b5e9-3c8d53c80021">


Making these adjustments ensures a cleaner structure and a more intuitive user flow. 
Happy coding! 😊🚀